# The code will output an error because the variable 
# can only be called within the scope of the function